cpp/log/cert.cc

```diff
@@ -357,20 +359,22 @@ StatusOr<bool> Cert::HasExtendedKeyUsage(int key_usage_nid) const {
     return util::Status(Code::INTERNAL, "NID lookup failed");
   }
 
-  void* ext_struct;
-  Status status = ExtensionStructure(NID_ext_key_usage, &ext_struct);
+  const StatusOr<void*> status = ExtensionStructure(NID_ext_key_usage);
```

Comment:
Done. Renamed to ext_status as we're only checking presence at this point. The structure is pulled out below.